src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBulkCSVFileRecord.java

```diff
@@ -189,15 +193,18 @@ private void initFileReader(InputStreamReader sr, String encoding, String demlim
         if (firstLineIsColumnNames) {
             currentLine = fileReader.readLine();
             if (null != currentLine) {
-                columnNames = currentLine.split(delimiter, -1);
+                columnNames = (escapeDelimiters && currentLine.contains("\""))
```

Comment:
We should keep the limit argument (-1) to the currentLine.split method to preserve the spacing at the end.

Reply:
Since we're going to ignore spaces outside double quotes, this code can stay as it is.